core/src/main/java/org/apache/calcite/sql/type/SqlTypeFamily.java

```diff
@@ -59,6 +59,7 @@ public enum SqlTypeFamily implements RelDataTypeFamily {
   BOOLEAN,
   INTERVAL_YEAR_MONTH,
   INTERVAL_DAY_TIME,
+  INTERVAL_WEEK_DAY,
```

Comment:
Why do we need an INTERVAL_WEEK_DAY SqlTypeFamily? Does INTERVAL_WEEK have to belong to a typefamily, so we created this one?

Reply:
Because to me, having INTERVAL_DAY inside of the WEEK_INTERVAL_TYPES SqlTypeNames is a little strange.

Reply:
Yea I agree with you. Changed it so WEEK_INTERVAL_TYPES contains only INTERVAL_WEEK.